server/utils/assessments/placementRequestData.ts

```diff
@@ -61,55 +81,33 @@ export const criteriaFromMatchingInformation = (
   const essentialCriteria = [] as Array<PlacementCriteria>
   const desirableCriteria = [] as Array<PlacementCriteria>
 
-  placementRequirements.forEach(requirement => {
+  if (matchingInformation.apType !== 'normal') {
+    essentialCriteria.push(matchingInformation.apType)
+  }
+
+  if (matchingInformation.mentalHealthSupport.length) {
```

Comment:
Any reason for this to not just be `if (matchingInformation.mentalHealthSupport.length) {`?

Reply:
If it's an empty string, it will still return as a truthy value, so we have to use `length`

Reply:
Empty string is falsey in JS, right?

Reply:
I feel like I forget this every 6 months, then I'm like "really???" 😛